packages/react-ui-components/src/ToggablePanel/toggablePanel.js

```diff
@@ -231,6 +231,7 @@ export class Header extends PureComponent {
                     {children}
                 </HeadlineComponent>
                 <IconButtonComponent
+                    tooltipLabel="Toogle the panel"
```

Comment:
I think this one is actually hard to localize; as we do not have a dependency from react-ui-components to the i18n package... So I'd leave it as is for now.